.github/workflows/pull-request-automation.yml

```diff
@@ -1,6 +1,7 @@
 on:
   pull_request:
     types: [opened, closed]
```

Comment:
We could revert the changes from #19742, since the only task which relied on `closed` is now using a different event.

Reply:
Done